client/src/components/SizeDropDown.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import styled from 'styled-components';
+import { DropDownList, DropDownItem } from '../styling.jsx';
+import DropDownHeader from './DropDownHeader.jsx';
+
+class SizeDropDown extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      dropDownOpen: false,
+    };
+    this.toggleDropDown = this.toggleDropDown.bind(this);
+  }
+
+  toggleDropDown() {
+    this.setState({ dropDownOpen: !this.state.dropDownOpen})
+  }
+
+  render() {
+    return (
+      <div className={this.props.className}>
+        <DropDownHeader toggleDropDown={this.toggleDropDown}/>
+        <DropDownList dropDownOpen={this.state.dropDownOpen}>
+          {this.props.sizes.map((size) => {
+            return <DropDownItem>{size}</DropDownItem>
```

Comment:
Nice code here! Easy to understand what's going on thanks to your variable and component names. 🥇